src/contexts/navigationContext.tsx

```diff
@@ -81,7 +81,7 @@ export interface NavigationInterface {
 
 export type NavigationProviderProps = {
   /**
-   * Strict navigations guard toggle. If configure to `true`, user can only navigate
+   * Strict navigations guard toggle. If configure to `true`, user cannot navigate
    * to immediate next or previous step; when configure to `false`, user can navigate to any
    * steps at any time. *Default* to `true`.
```

Comment:
it should be `user can navigate to current and previous step`

Reply:
If `strict` mode enabled, user shouldn't be able to navigate to next and previous right ?

Reply:
i believe user should still be allowed to go back to steps that have completed, otherwise they are stuck in the current steps that they are in

Reply:
True, I forgot about the current and previous step